.gitignore

```diff
@@ -38,7 +38,7 @@ bower_components
 build/Release
 
 # Dependency directories
-node_modules/
+examples/node_modules/
```

Comment:
**suggestion:** the .idea folder is usually added to the .gitignore file, since its developer environment specific